tests/run-make/extern-flag-disambiguates/rmake.rs

```diff
@@ -0,0 +1,30 @@
+//@ ignore-cross-compile
+
+use run_make_support::{cwd, run, run_in_tmpdir, rustc};
+
+// Attempt to build this dependency tree:
+//
+//	A.1   A.2
+//	 |\    |
+//	 | \   |
+//        B  \  C
+//         \ | /
+//          \|/
+//           D
+//
```

Comment:
On Github, it appears as if A.1 and A.2 are offset from B and C, which is not the same as the original Makefile graph, but it could just be a rendering issue. Maybe it has something to do with tab characters?

Reply:
Yes, GitHub will render the offset according to your tab settings, we should use only spaces.

Reply:
oops, thanks!